app/views/filters/index.html.erb

```diff
@@ -1,4 +1,8 @@
-<% title _("Filters") %>
+<% if @role
+     title _("Filters for role") + " " + @role.to_s
```

Comment:
To translate this correctly, it should be: `title _("Filters for role %s") % @role.to_s` which will allow translators to move the %s around as they like (http://projects.theforeman.org/projects/foreman/wiki/Translating#Translating-for-developers has more tips).